worker/src/queues/notify/workers/notifyFailureHandler.ts

```diff
@@ -0,0 +1,70 @@
+import pino from "pino";
+import { Job } from "pg-boss";
+import { NotifyClient, Status, ErrorResponse, GetNotificationsResponse } from "notifications-node-client";
+import config from "config";
+import { NOTIFY_FAILURE_ERRORS } from "./errors";
+
+const queue = "NOTIFY_FAILURE_CHECK";
+const worker = "notifyFailureHandler";
+
+const logger = pino().child({
+  queue,
+  worker,
+});
+
+const notifyClient = new NotifyClient(config.get<string>("Notify.apiKey"));
+
+export async function notifyFailureHandler(job: Job) {
+  const jobId = job.id;
+  logger.info({ jobId }, `received ${worker} job`);
+  try {
+    const responses = await getFailureResponses();
+    const failures = responses
+      .flatMap((response, index) => {
+        if (response.status !== 200) {
+          logger.warn(`Could not fetch email failures for status: ${failureStatuses[index]}`);
+          return [];
+        }
+        return (response.data as GetNotificationsResponse).notifications;
+      })
+      .filter((failure) => failure);
+    if (failures.length > 0) {
```

Comment:
Possibly a redundant message depending on my comment on `getFailureResponses` - but this should return early if the array is empty. Generally - write the guard clause which results the least nesting. 

```
if(failures.length === 0) {
  logger.info(...)
  return
}

// everything else unnested
```